src/key_reset.c

```diff
@@ -925,9 +931,15 @@ PEP_STATUS create_standalone_key_reset_message(PEP_SESSION session,
 
     message* output_msg = NULL;
     
-    status = encrypt_message(session, reset_msg, NULL,
-                             &output_msg, PEP_enc_auto,
-                             PEP_encrypt_flag_key_reset_only | PEP_encrypt_flag_force_no_attached_key);
+    PEP_encrypt_flags_t flags =
+       PEP_encrypt_flag_key_reset_repropagate
```

Comment:
I see we use the flag PEP_encrypt_flag_key_reset_repropagate every time on key reset, this means we will keep the old fpr in the "from" filed in all key reset messages (which I thought we were doing already).